QA_Robot_Practice/TestSuites/Clarabri-10001_PythonLoginTest.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Resource          Resources/Resources.robot
+Resource          ../utils/RandomWordsUtil.robot
+Resource          ../globalConfig/GlobalSettings.robot
```

Comment:
Change ../globalConfig/GlobalSettings.robot to  ../globalConfig/GlobalSettings.py